tests/backup/backup_helper.go

```diff
@@ -1497,12 +1502,34 @@ func DeletePodWithLabelInNamespace(namespace string, label map[string]string) er
 	}
 
 	// fetch the newest set of pods post wait for pods to come up
-	pods, err = core.Instance().GetPods(namespace, label)
+	if ignoreLabel {
```

Comment:
Add this as part of **podList** instead of a separate call

Reply:
Based on offline discussion we decided to not go with this as it will be more time complex as we would need to run through the pod list twice hence. We'll revisit this code as the logic of the function isn't very clear. When we revist the code we'll fix the logic. 

For now going with the current logic until we revisit again